sqldef.go

```diff
@@ -13,6 +13,7 @@ import (
 
 type Options struct {
```

Comment:
Can we just let it have only `DesiredDDLs` and `CurrentDDLs`? Having both `DesiredFile` and `DesiredDDLs` seems confusing.

Reply:
Trying to make a diff as small as possible is a bad habit of mine.